src/pages/form-validation/password/index.tsx

```diff
@@ -0,0 +1,472 @@
+
+    return (
+        <div
+            style={{
+                backgroundColor: '#E5E5E5',
+                color: theme.palette.text.primary,
+                minHeight: '100vh',
+                display: 'flex',
+                flexDirection: 'column',
+            }}
+        >
+            <AppBar data-cy="pxb-toolbar" position={'sticky'} classes={{ root: classes.appbarRoot }}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Change Password
+                    </Typography>
+                </Toolbar>
+            </AppBar>
+
+            <div className={classes.containerWrapper}>
+                <Card className={classes.container} elevation={4}>
+                    <Typography variant={'h6'} className={classes.sectionHeader}>
+                        Change Password
+                    </Typography>
+                    <Typography variant={'body1'}>
+                        Password must be at least 8 characters long, contain at least one uppercase character, one
+                        lowercase character, one number, and one special character.
+                    </Typography>
+
+                    <Divider className={clsx(classes.divider, classes.topDivider)} />
+
+                    <div className={classes.formOverflow}>
+                        <form>
+                            <TextField
+                                id={'currentPassword'}
+                                label={'Old Password'}
+                                type={showCurrentPassword ? 'text' : 'password'}
+                                onChange={onCurrentPasswordChange}
+                                value={currentPassword}
+                                error={Boolean(currentPasswordError)}
+                                onBlur={validateCurrentPassword}
+                                required
+                                fullWidth
+                                variant={'filled'}
+                                InputProps={{
+                                    endAdornment: (
+                                        <InputAdornment position={'end'}>
+                                            <IconButton
+                                                style={{ height: 36, width: 36 }}
```

Comment:
Instead of inline style like this, you can do `edge={'end'}`

Reply:
Using just the `edge` prop made the icon ripple look too large for the form field, so I just add a class instead.